include/collective.hpp

```diff
@@ -61,8 +61,6 @@ class Collective
   void run(Timeline &, std::unique_ptr<Phase> &&);
 
   Collective(FaultInjector *faults);
-  Collective(std::initializer_list<int> selected,
```

Comment:
The reason for `selected` to exist is to make it easier for testing (unit tests are broken for some time, though). Don't remove this constructor.